spec/features/users/login_spec.rb

```diff
@@ -28,6 +27,42 @@
       expect(page).to have_content("Welcome go@foogle.com")
     end
 
+    it "won't login if password does not match" do
+
+      new_user = create(:regular_user, email: "yo@gmail.com", password: "123password", password_confirmation: "123password")
+
+      visit '/merchants'
+      click_on 'Log In'
+
+      expect(current_path).to eq("/login")
+
+      fill_in :email, with: "yo@gmail.com"
+      fill_in :password, with: "notsecure123"
+      fill_in :password_confirmation, with: "notsecure123"
+      click_button "Log In"
+
+      expect(current_path).to eq("/login")
+      expect(page).to have_content("Sorry Invalid Password or Email.")
+    end
+
```

Comment:
The sad path testing for both bad email and bad password is awesome! Great coverage.